src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -676,27 +985,32 @@ def svm_help(schema_madlib, message, is_svc, **kwargs):
 def svm(schema_madlib, source_table, model_table,
         dependent_varname, independent_varname, kernel_func,
         kernel_params, grouping_col, params, is_svc,
-        verbose, detect_novelty=False, **kwargs):
+        verbose, add_intercept=False, **kwargs):
```

Comment:
It occurs to me that `add_intercept` will always be `False` here since we are not exposing it to the users and no client code is calling this function. A relevant question to think about is whether we should add `fit_intercept` to the `params` and discourage the use of `array[..., 1]`.